openhtf/output/callbacks/mfg_inspector.py

```diff
@@ -121,7 +121,7 @@ def _populate_header(record, testrun):
     attachment = testrun.info_parameters.add()
     attachment.name = 'config'
     attachment.value_binary = json.dumps(
-        record.metadata['config'], sort_keys=True, indent=4)
+        record.metadata['config'], sort_keys=True, indent=4).encode('utf-8')
```

Comment:
Maintainers, PTAL at this too, do you want these json dumps to be unicode or byte strings?

Reply:
config should generally be YAML text from a file so utf-8 makes sense as an encoding here also.